ansible_collections/axonops/configuration/plugins/modules/alert_rule.py

```diff
@@ -144,29 +144,33 @@ def run_module():
     if error is not None:
         error_message = "Error occurred accessing alert URL: " + alerts_url + str(error)
         module.fail_json(msg=error_message)
-        return
-
-    dash_templates, error = axonops.do_request(
-        f"/api/v1/dashboardtemplate/{org}/{axonops.get_cluster_type()}/{cluster}")
-    if error is not None:
-        error_message = ("Error occurred fetching AxonOps dashboard template: "
-                         + f"/api/v1/dashboardtemplate/{org}/{axonops.get_cluster_type()}/{cluster}"
-                         + str(error))
-        module.fail_json(msg=error_message)
-        return
 
-    # Find the referenced dashboard by name
-    new_dash = find_by_field(dash_templates.get('dashboards'), 'name', module.params['dashboard'])
-    if not new_dash:
-        module.fail_json(msg=f"Could not find dashboard '{module.params['dashboard']}' in AxonOps")
-        return
-
-    # Find the referenced chart in the dashboard
-    new_charts = find_by_field(new_dash.get('panels'), 'title', module.params['chart'])
-    new_chart = None
+    dashboard_template_api = f"/api/v1/dashboardtemplate/{org}/{axonops.get_cluster_type()}/{cluster}?dashver=2.0"
+    dashboard_template_api_v1 = f"/api/v1/dashboardtemplate/{org}/{axonops.get_cluster_type()}/{cluster}"
+
+    for url in [dashboard_template_api, dashboard_template_api_v1]:
+        dash_templates, error = axonops.do_request(url)
+        if error is not None:
+            error_message = ("Error occurred fetching AxonOps dashboard template: "
+                             + url
+                             + str(error))
+            module.fail_json(msg=error_message)
+
+        # Find the referenced dashboard by name
+        new_dash = find_by_field(dash_templates.get('dashboards'), 'name', module.params['dashboard'])
+        if not new_dash:
```

Comment:
Instead of failing if the dashboard is not found this should use `continue` to fall back to the next URL in the list